anyric/activo-web
# #158496205 Add test coverage badge

**What does this PR do?**

This PR add test coverage badge to the ReadMe

**Description of Task to be completed?**

Configure test coverage reporting for activo web on CircleCi

**How should this be manually tested?**

Make some changes and commit to github and select the branch you made the changes on to see the test coverage on build is completed on CircleCi

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#158496205](https://www.pivotaltracker.com/story/show/158496205)

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
